main/src/main/java/org/mobilitydata/gtfsvalidator/validator/DuplicateRouteNameValidator.java

```diff
@@ -49,7 +49,7 @@ public void validate(NoticeContainer noticeContainer) {
             route -> {
               if (route.hasRouteShortName() && route.hasRouteLongName()) {
                 if (routeByShortAndLongName.containsKey(
-                    route.routeShortName() + route.routeLongName())) {
+                    route.routeShortName() + route.routeLongName() + route.routeType())) {
```

Comment:
When you have a composite key of this many values, I'd suggest creating a single private method to construct the key - that will enforce that it always has the correct components in the correct order.